src/ios/SegmentCordovaPlugin.m

```diff
@@ -230,4 +231,14 @@ - (void)reset:(CDVInvokedUrlCommand*)command {
     [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void)isInitialized:(CDVInvokedUrlCommand*)command {
+    CDVPluginResult* pluginResult = nil;
+
+    pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK
+                                    messageAsInt:(isInitialized ? 1 : 0)];
```

Comment:
Cordova doesn't support messages as a boolean, so that's why it's convert to a int here (same above for android)